docs/authentication/social-connections/custom-provider.mdx

```diff
@@ -0,0 +1,78 @@
+---
+description: Configure a custom OpenID Connect (OIDC) compatible authentication provider for your Clerk application.
+---
+
+# OAuth with a custom auth provider
+
+<Callout type="warning">
+  Custom auth providers are in beta. Please [contact support](/contact/support) to get your application whitelisted for
+  this feature.
+</Callout>
+
+Clerk allows you to configure custom OpenID Connect (OIDC) compatible authentication providers for your application. This guide walks you through the steps to set up a custom OAuth provider.
+
+## Prerequisites
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "A Clerk application is required.",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "An OIDC identity provider is required.",
+      link: "https://openid.net/specs/openid-connect-core-1_0.html",
+      icon: "user-circle",
+    }
+  ]}
+>
+  - Configure custom Auth provider for your Clerk application.
+</TutorialHero>
```

Comment:
```suggestion
```

Reply:
Seems related to this: https://github.com/clerk/clerk-docs/pull/1321#discussion_r1698857252